### ISSUE TITLE
Missing button/context menu to close minibrowser side area

### ISSUE THREAD

**Issue by joazvsoares** [eclipse-theia/theia#6496]
### Description
Currently, we don't have a very intuitive way to close the minibrowser widget, one has to use the context menu, i.e. right click on the icon in the side area.

It would be nice to have the close icon there and also the context menu should work from the title the same way as in the main area.

![image](https://user-images.githubusercontent.com/5097842/68218055-7a2cb780-ffc2-11e9-9a25-7cb499bbc72d.png)

### Reproduction Steps
1. Go to > Preview: Open Url 
2. Type any URL
3. There is no close icon and also no context menu

**OS and Theia version:**
Theia 0.8.0

**Comment by akosyakov** [comment]
Generally, it is not nice that a widget toolbar in side areas does not provide the same actions like the toolbar in the main area. For first time user it is confusing to find.

**Comment by akosyakov** [comment]
@eclipse-theia/eclipse-theia Does anyone oppose an idea or have a better idea? If not i could guide how to implement it.

**Comment by vince-fugnitto** [comment]
I guess it's just something that is understood the more you use the IDE regularly.
It will only be confusing at the beginning if at all. 
If we were to fix the issue however, it should be for all widgets, not only the `preview`.


<div align='center'>

<img width="547" alt="Screen Shot 2019-11-06 at 3 07 38 PM" src="https://user-images.githubusercontent.com/40359487/68333764-38c20800-00a7-11ea-9b3c-834de678e399.png">


</div>

**Comment by vince-fugnitto** [comment]
> @eclipse-theia/eclipse-theia Does anyone oppose an idea or have a better idea? If not i could guide how to implement it.


Perhaps for sidebar widgets, right-clicking the tabbar title (`preview` label in this case), can trigger a context menu with the ability to close? Not sure how much more intuitive this is either.

<div align='center'>

<img width="407" alt="Screen Shot 2019-11-06 at 3 11 52 PM" src="https://user-images.githubusercontent.com/40359487/68334043-cc93d400-00a7-11ea-979d-bdf0bd674c5d.png">


</div>

**Comment by akosyakov** [comment]
> Perhaps for sidebar widgets, right-clicking the tabbar title (preview label in this case), can trigger a context menu with the ability to close? Not sure how much more intuitive this is either.

Yes, I also was thinking about the separate action group in the toolbar context menu.

I'm a bit unsure about close button for all widgets. It makes sense for editors and previews, but for views it does not feel good somehow. I have not seen such button in side areas in other IDEs.

**Comment by joazvsoares** [comment]
@akosyakov have you decided on the approach we should take for this issue? Maybe I can collaborate if you can provide some guidance on the implementation.

**Comment by akosyakov** [comment]
@joazvsoares I think everybody agree that having a group in the context menu will be nice. Let's do it first. I will investigate what should be done and write down here.

Adding a close button is not so clear, it makes sense for some widgets but not others, i.e. views.

**Comment by akosyakov** [comment]
That's a bit tricky because of view containers, i.e. the navigator and scm, since they are hijacking the context menu rendering for tab bars already.

But if you are up to it there are 2 steps:
- Customize `SidePanelToolbar` to render `SHELL_TABBAR_CONTEXT_MENU` context menu for the title container: https://github.com/eclipse-theia/theia/blob/71c11c36171398975779c925bbc0ca3910189853/packages/core/src/browser/shell/side-panel-toolbar.ts#L25.
  - You can look at the standard tab bar for the inspiration: https://github.com/eclipse-theia/theia/blob/71c11c36171398975779c925bbc0ca3910189853/packages/core/src/browser/shell/tab-bars.ts#L397
- Change the view container to contribute its action group to `SHELL_TABBAR_CONTEXT_MENU` when necessary instead of [rendering own](https://github.com/eclipse-theia/theia/blob/71c11c36171398975779c925bbc0ca3910189853/packages/core/src/browser/view-container.ts#L118-L124). Such action group should go before other actions in the context menu.
